src/components/propertydetails/Review.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import "./propertydetails.css";
+
+function Review() {
+  return (
+    <div className="review-container">
+      <div className="email-agent-container">
+        <span className="email-agent-header">
+          <h4>More about Property</h4>
+        </span>
+        <div className="contact-agent-wrapper">
+          <div className="input-container">
+            <i className="fa fa-user form-icon"></i>
+            <input className="field" type="text" placeholder="Username" />
+          </div>
+          <div className="input-container">
+            <i className="fa fa-envelope form-icon"></i>
+            <input className="field" type="tel" placeholder="Email" />
+          </div>
+          <div className="input-container">
+            <i className="fa fa-phone form-icon"> </i>
+            <input className="field" type="email" placeholder="Phone" />
```

Comment:
Looks like there's a mix up here, type of email and placeholder phone. What are we expecting here?

Reply:
> Looks like there's a mix up here, type of email and placeholder phone. What are we expecting here?

would change that